articles/cost-management-billing/manage/link-partner-id-power-apps-accounts.md

```diff
@@ -34,7 +34,7 @@ When you have access to your customer's resources, use the Azure portal, PowerSh
 
 1. Sign in to the [Azure portal](https://portal.azure.com).
 1. Go to [Link to a partner ID](https://portal.azure.com/#blade/Microsoft_Azure_Billing/managementpartnerblade) in the Azure portal.
-1. Enter the [Microsoft Partner Network](https://partner.microsoft.com/) ID for your organization. Be sure to use the  **Associated MPN ID**  shown on your partner profile.  
+1. Enter the [Microsoft Partner Network](https://partner.microsoft.com/) ID for your organization. Be sure to use the  **Associated MPN ID**  shown on your partner center profile. This is typically known as your [Partner Location Account MPN ID](https://docs.microsoft.com/en-us/partner-center/account-structure).  
```

Comment:
```suggestion
1. Enter the [Microsoft Partner Network](https://partner.microsoft.com/) ID for your organization. Be sure to use the  **Associated MPN ID**  shown on your partner center profile. This is typically known as your [Partner Location Account MPN ID](/partner-center/account-structure).  
```

Reply:
#hold-off 
@JasonWHowell Oh, good catch Jason!
@AkanshaGawade Please look at Jason's proposed change above and make the update. Otherwise, the pull request reviewer will reject this PR until the change is made.